docker/Dockerfile-dev

```diff
@@ -1,14 +1,15 @@
 ARG dev_cycle=develop
-ARG registry=lsstts
-FROM ${registry}/develop-env:${dev_cycle}
+FROM lsstts/develop-env:${dev_cycle}
 
 WORKDIR /usr/src/love/
-COPY requirements.txt .
+COPY . .
```

Comment:
Out of curiosity, what does this line do? If it is supposed to copy one or more files then please replace that with explicit `COPY` lines so it is clear what it is supposed to copy and doesn't copy any unnecessary or unwanted files.

Reply:
Also we kind of use everything on the repo, the only files that are not used are the Dockerfiles themselves, however we still use a couple `sh` scripts on that folder, so that's why I thought on including it completely.

Reply:
Thanks for explaining!